app/models/field.rb

```diff
@@ -2,7 +2,7 @@
 
 class Field < ApplicationRecord
   belongs_to :calculator
-  before_create :set_selector
+  before_create :manage_selector
```

Comment:
`manage_selector ` method does not exist

Reply:
Thanks. Done